debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/RecordMakers.java

```diff
@@ -173,7 +173,7 @@ public boolean assign(long tableNumber, TableId id) {
         TableSchema tableSchema = schema.schemaFor(id);
         if (tableSchema == null) return false;
 
-        String topicName = topicSelector.getTopic(id);
+        String topicName = tableSchema.getParentSchemaName();
```

Comment:
Hmm ok. With these changes, the value of `tableSchema.getParentSchemaName()` basically boils down to `topicMapper.getTopicName()`. In the case of a `ByDatabaseTopicMapper`, many tables will share the same topic name, and thus the same Envelope schema name. Is that ok, or do the envelope schema names need to be distinct per table?